app/main.py

```diff
@@ -2,0 +1,18 @@
+def copy_file(command: str) -> None:
+    file_names = command.split()
+    if len(file_names) != 3 or file_names[0] != "cp":
+        return
+
+    if file_names[1] == file_names[2]:
+        return
```

Comment:
It might be helpful to inform the user that the source and destination files are the same, which is why the operation is not performed.